lstchain/scripts/lstchain_dl1_to_dl2.py

```diff
@@ -1,10 +1,8 @@
 #!/usr/bin/env python3
 __doc__ = """
 Run the DL1 to DL2 step: Pipeline for the reconstruction of Energy, disp and gamma/hadron
-separation of events stored in a DL1 file.
-- Input: DL1 files and trained Random Forests.
-- Output: DL2 data file.
-
+separation of events stored in a DL1 file, taking DL1 files and trained Random Forests as input 
```

Comment:
Put "DLn 'data' file" or "DLn file" for both DL1 and DL2?
May be a bit easier to read if you split the sentence:
"... in a DL1 file. Takes DL1 files and...as input. Outputs DL2 files."
or 
"... in a DL1 file. Takes DL1 files and...as input and outputs DL2 files."

Reply:
done